gatsby-config.js

```diff
@@ -1,38 +1,54 @@
-// require("dotenv").config();
+const path = require('path')
 
 // Gatsby config
 module.exports = {
-  pathPrefix: "/",
-  plugins: [
-    {
-      resolve: "gatsby-source-filesystem",
-      options: {
-        name: "transcripts",
-        path: `${__dirname}/src/transcripts/`
-      }
-    },
-    {
-      resolve: "gatsby-source-filesystem",
-      options: {
-        name: "mdPages",
-        path: `${__dirname}/src/md-pages/`
-      }
-    },
-    {
-      resolve: "gatsby-plugin-react-svg",
-      options: {
-        path: `${__dirname}/src/assets/`,
-        rule: {
-          include: [/icons/, /svg\//]
-        }
-      }
-    },
-    {
-      resolve: "gatsby-transformer-remark",
-      options: {
-        plugins: [`gatsby-remark-autolink-headers`]
-      }
-    },
-    "gatsby-plugin-styled-components"
-  ]
-};
+	siteMetadata: {
```

Comment:
Are these tabs I spy 👀

Reply:
Hahah yeah... in my defence it's the default prettier config so I didn't think about it! 😛 

Don't think there's much of a difference between the two, but what persuaded to move from spaces -> tabs by default was when someone pointed out that tab width is adjustable while space width is not. Therefore tabs mean that _everyone_ can adjust the indentation to suit their preferences, while spaces mean that only the people who like double-space get to read code in their preferred formatting. 😉

Reply:
I agree with that, but I hate how GitHub renders tabs by default. Not gonna lie if that one thing changed (or was configurable) I'd probably hop (back) on the tab train.

Reply:
Oh yeah true - couldn't live without [Refined Github](https://github.com/sindresorhus/refined-github/) for so many details like that tbh... 🙈